cafebabel/users/models.py

```diff
@@ -21,6 +20,10 @@ class UserProfile(db.EmbeddedDocument, UploadableImageMixin):
     about = db.StringField()
     old_pk = db.IntField()  # In use for migrations (references in articles).
 
+    meta = {
+        'indexes': ['old_pk']
```

Comment:
Should the meta be on the Embedded class or the Article class?
https://stackoverflow.com/a/30455984/328117